Makefile

```diff
@@ -0,0 +1,31 @@
+COVERAGE_ANNOTATION=coverage_annotations
+TEST_CMD=manage.py test orchestra beanstalk_dispatch
```

Comment:
probably fine if we don't call out the projects, so we can add more w/o having to update this.